tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -188,9 +188,9 @@ Status ModularFileSystem::GetChildren(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-Status ModularFileSystem::GetMatchingPaths(
-    const std::string& pattern,
-    std::vector<std::string>* result /*, TransactionToken* token */) {
+Status ModularFileSystem::GetMatchingPaths(const std::string& pattern,
+                                           std::vector<std::string>* result,
+                                           TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `result` which will hold the results